src/common/stats/test/StatsManagerBenchmark.cpp

```diff
@@ -62,6 +62,39 @@ BENCHMARK(add_histogram_value_8t, iters) {
 
 BENCHMARK_DRAW_LINE();
 
+const int32_t counter = StatsManager::registerStats("Counter");
```

Comment:
Do we need to update the benchmark numbers?

Reply:
I thinks so.